lib/private/Setup/PostgreSQL.php

```diff
@@ -152,11 +152,11 @@ private function createDBUser(IDBConnection $connection) {
 			}
 
 			// create the user
-			$query = $connection->prepare("CREATE USER " . addslashes($this->dbUser) . " CREATEDB PASSWORD '" . addslashes($this->dbPassword) . "'");
-			$query->execute();
+			$query = $connection->prepare('CREATE USER :user CREATEDB PASSWORD :pass');
```

Comment:
```suggestion
			$query = $connection->prepare('CREATE USER :user CREATEDB PASSWORD \':pass\'');
```

or with double quotes for the sql statment and single quotes for pass.

Reply:
It's a prepared statement, so the quotes are added automatically.

I believe the issue is that the username is getting quoted like a normal string but actually requires a different type of quoting.

Reply:
> It's a prepared statement, so the quotes are added automatically.

Right ;)

Reply:
I think https://stackoverflow.com/a/18901826/433558 is the answer; I'll switch `:user` out for `pg_escape_identifier` as soon as I get a chance.